### PR TITLE
Update writing-day.rst

### PR DESCRIPTION
**Pull request by firstladymb**
Adding The Blog Writing Room

----
:books: Documentation preview :books:: https://writethedocs-www--2015.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/atlantic/2023/writing-day.rst [hunk 1]
```diff
@@ -67,3 +67,20 @@ together and offer each other some feedback on the important documentation
 we're using to move our careers forward.
 
 Let's help each other get ready for the job fair!
+
+The Blog Writing Room: Blogging for technical writers and documentarians
```
Comment (rosewms) [inline docs/conf/atlantic/2023/writing-day.rst]: I'd recommend shortening the title to "The Blog Writing Room" - the audience is implied and you mention it in your second sentence.

File: docs/conf/atlantic/2023/writing-day.rst [hunk 2]
```diff
@@ -67,3 +67,20 @@ together and offer each other some feedback on the important documentation
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Struggling to get into blogging? Our project, The Blog Writing Room, is designed to 
+give technical writers and documentarians a space to come together to work on writing 
+blog posts in a supportive and collaborative environment.
```
Comment (rosewms) [inline docs/conf/atlantic/2023/writing-day.rst:76]: Suggested edit to your second sentence: 

> "The Blog Writing Room is designed to create a supportive and collaborative environment for documentarians and tech writers to work on blog posts."

I would move your Writing Day goals under your first paragraph.

File: docs/conf/atlantic/2023/writing-day.rst [hunk 3]
```diff
@@ -67,3 +67,20 @@ together and offer each other some feedback on the important documentation
...
+
+Join us in the morning and/or the afternoon session for fireside roundtable discussions 
+followed by blog writing workhops where we can help each other brainstorm topics, put
+pen to paper, and figure out how to share our ideas, tips, and tricks with the community.
```
Comment (rosewms) [inline docs/conf/atlantic/2023/writing-day.rst]: @firstladymb I know you have an agenda that you plan to link, I'd recommend referencing the roundtable topics in this overview so people don't have to click through. 

I edited and reframed your content to make it a bit more action oriented. It's predominately suggestions based on how folks responded to previous Writing Day projects we posted. It's totally optional and you can decide if you want to implement it.

"You're welcome to join us any time throughout Writing Day. To make the most of our time together, we have planned two round table discussions and two writing workshops:

- Roundtable 1: "[title]"
- Blog writing workshop
- Writing Day snack break
- Roundtable 2: "[title]"
- Blog writing workshop

The workshops are a space for everyone to brainstorm and encourage each other to proverbially "put pen to paper." We can't wait to hear your ideas, tips, and tricks - no matter the topic. We're excited to see you share your knowledge with the Write the Docs community and beyond using blog posts!"

**Review by firstladymb — COMMENTED** [review]
Excellent feedback! I think I figured out how to make the updates and will send it back over for approval.

**Review by rosewms — APPROVED** [review]
Looks good to me, I'm going to merge it so we can get it live. @firstladymb you can make a new PR with the link the the agenda later, tag me in it and I'll get it live.

**Pull request merged**